autobuild.xml

```diff
@@ -99,18 +99,28 @@
       </map>
       <key>boost</key>
       <map>
+        <key>copyright</key>
```

Comment:
Pretty sure the next one doing library updates is going to shuffle all these keys to the end again... :joy:

Reply:
It depends on the autobuild version, but yes, it's a known problem.